src/plots/layout_attributes.js

```diff
@@ -292,6 +292,16 @@ module.exports = {
             'Sets the background color of the plotting area in-between x and y axes.'
         ].join(' ')
     },
+    axesconvertnumeric: {
+        valType: 'boolean',
+        dflt: true,
+        role: 'info',
+        editType: 'calc',
+        description: [
+            'Determines whether or not a numeric string in axes of the plot may be',
```

Comment:
```suggestion
            'Determines whether or not a numeric string in trace data may be',
```

Reply:
Also something about "this is the default value, and may be overridden for individual axes"